src/RevProc.cc

```diff
@@ -1682,6 +1682,8 @@ void RevProc::MarkLoadComplete(const MemReq& req){
 }
 
 bool RevProc::ClockTick( SST::Cycle_t currentCycle ){
+  uint32_t RV32b[_REV_NUM_REGS_];
+  uint64_t RV64b[_REV_NUM_REGS_];
```

Comment:
In `RevRegFile.h`, we used anonymous `union`s to join the `RV32` and `RV64` arrays.

Please see https://github.com/tactcomplabs/rev/pull/138

Reply:
Good point, changed. Thank you!